src/PHPMailer.php

```diff
@@ -1994,7 +1994,7 @@ public function setLanguage($langcode = 'en', $lang_path = '')
         ];
         if (empty($lang_path)) {
             // Calculate an absolute path so it can work if CWD is not here
-            $lang_path = __DIR__ . DIRECTORY_SEPARATOR . 'language' . DIRECTORY_SEPARATOR;
+            $lang_path = __DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'language' . DIRECTORY_SEPARATOR;
```

Comment:
Rather than building a path string containing a relative path that will need to be interpreted, I think it's probably better to calculate the parent folder directly, something like:

    $lang_path = dirname(__DIR__) . DIRECTORY_SEPARATOR . 'language' . DIRECTORY_SEPARATOR;

Reply:
Done.